### ISSUE TITLE
rename sections in About

### ISSUE THREAD

**Issue by bertsky** [OCR-D/ocrd-website#243]
I really like the new sections in ocr-d.de's `About` header frame list, but would suggest renaming as follows:

- `Get in touch!` → `Contacts` (easier to find at a glance; less ambiguous – _community platforms_ can also get you in touch)
- `Platforms` → `Community Platforms` or just `Community` (just _platforms_ could also mean operating systems, hardware devices...)
- `Initial tests` → `Pilot Study` or `Initial User Tests`, perhaps even `2019 Pilot Study` (software had already been tested by developers on various materials)
- `The OCR-D-project` → `About the OCR-D Project` or just `Introduction`
- `OCR-D Phase II` → `Phase II Projects`
- `OCR-D Phase III` → `Phase III Projects`

Analogously in German.

(Also, I think the introductory text under `The OCR-D-project` should be updated to reflect transition to phase III and current status. Plus, IMHO all the valuable entries under `About` list, not only that introductory text, should also be directly visible on the German and English landing pages – just like all the entries under `Technical Resources` and `Guides & Info` are.)

**Comment by bertsky** [comment]
> What do you think about the new landing page (https://ocr-d.de/de/ / https://ocr-d.de/en/) ?

Very nice! That should also help search engines and web archive. 
(Some subpages are still not referenced outside the navbar though: projects2, publications, data, pilot, survey, impress. Perhaps you can reference them from other pages?)

**Comment by lena-hinrichsen** [comment]
> > (Some subpages are still not referenced outside the navbar though: projects2, publications, data, pilot, survey, impress. Perhaps you can reference them from other pages?)

Yes, I focused on those subpages which seem to be most important. Maybe I can reference the others in `about` and `phase2` ? The imprint could be added to the footer.

**Comment by lena-hinrichsen** [comment]
done, feedback is appreciated :)
